src/slider/types.js

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {OverrideT} from '../helpers/overrides';
+import {} from './constants';
+
+export type OverridesT = {
+  Root?: OverrideT<*>,
+  Axis?: OverrideT<*>,
+  Tick?: OverrideT<*>,
+  TickBar?: OverrideT<*>,
+  Thumb?: OverrideT<*>,
+};
+
+export type PropsT = {
+  value: Array<*>,
+  range: Array<*>,
+  overrides?: OverridesT,
+  tabIndex: number,
+  error: boolean,
+  autoFocus: boolean,
+  disabled?: boolean,
+  $theme?: *,
```

Comment:
Why not `ThemeT` here?

Reply:
Also ThemeT isn't really part of the public props interface.

Reply:
fixed